packages/faceted-search/src/components/BasicSearch/BasicSearch.component.js

```diff
@@ -37,15 +37,16 @@ const BasicSearch = ({
 	customBadgesDictionary,
 	customOperatorsDictionary,
 	initialFilterValue,
+	useColors = false,
```

Comment:
Why have a conditional feature boolean here? I think we need to provide all badges supports by default.
It's better to modify the badgesDefinitions instead, to be sure that FacetedSearch is as dumb as possible.

Reply:
> I make a first read of the PR but i'm a little lost of this PR, it's not a new badge right? Can you link a public link to sketch? The one on the PR description is private

the point is to add colors to existing badges and new badges for TDS and TDP
![image](https://user-images.githubusercontent.com/58977230/92402951-ca1f6200-f130-11ea-9c77-c8a664c8d911.png)

I split the PR, this one is now about Badge component only, faceted search code will be in another one